lib/tasks/import.rake

```diff
@@ -5,9 +5,27 @@ require "gds_api/json_client"
 namespace :import do
   desc "Import all documents matching an organisation and document type from Whitehall Publisher, e.g. import:whitehall_migration[organisation-slug, document-type]"
   task :whitehall_migration, %i[organisation_slug document_type] => :environment do |_, args|
-    WhitehallMigration.create!(organisation_slug: args.organisation_slug,
-                               document_type: args.document_type,
-                               start_time: Time.zone.now)
+    ActiveRecord::Base.transaction do
+      WhitehallMigration.create!(organisation_slug: args.organisation_slug,
+                                 document_type: args.document_type,
+                                 start_time: Time.zone.now)
+
+      items_per_page = 100
+
+      1.step do |page_number|
```

Comment:
`step` provides an index (which I'm using as the page number) which we wouldn't get with `loop`.  I guess `until` would also do a similar job, but I'd need to increment the page number manually in each iteration.

I think although it's not written, I'm effectively testing it doesn't loop any more than twice since those are the only HTTP requests that are stubbed (a third request would have `page_number=3`.  But I'll see if there's some way I could explicitly test this.